packages/neuron-ui/src/components/DepositDialog/index.tsx

```diff
@@ -151,8 +160,10 @@ const DepositDialog = ({
             className={styles.depositValue}
             width="100%"
             field="depositValue"
-            value={localNumberFormatter(maxDepositValue ?? depositValue)}
+            value={isTyping ? maxDepositValue ?? depositValue : localNumberFormatter(maxDepositValue ?? depositValue)}
```

Comment:
`maxDepositValue ?? depositValue` can be assigned to a variable to make it clear